packages/cli/internal/pkg/cli/zipfile/compress_to_tmp.go

```diff
@@ -25,6 +27,14 @@ func CompressToTmp(srcPath string) (string, error) {
 }
 
 func writeToZipRecursive(writer *zip.Writer, rootPath string) error {
+	// Expand home directory path
+	usr, _ := user.Current()
+	dir := usr.HomeDir
+	if rootPath == "~" {
```

Comment:
Can you please add some tests for this? There are some existing examples you can extend although you will need to create a global function for `user.Current` and `usr.HomeDir` to test this (i.e. Like the mkDirTemp example here: https://github.com/aws/amazon-genomics-cli/blob/main/packages/cli/internal/pkg/aws/cdk/deploy_app.go#L9 ).

Reply:
Added unit tests.